dbt_copilot_helper/commands/bootstrap.py

```diff
@@ -66,6 +68,10 @@ def make_config(directory="."):
 
     click.echo(">>> Generating Copilot configuration files\n")
 
+    # add .copilot-helper-version file
+    copilot_version = string_version(get_copilot_versions()[0])
+    click.echo(mkfile(base_path, ".copilot-version-file", f"{copilot_version}"))
```

Comment:
As part of [this ticket](https://uktrade.atlassian.net/browse/DBTP-763) I've named the version file `.copilot-helper-version`

```suggestion
    click.echo(mkfile(base_path, ".copilot-helper-version", f"{copilot_version}"))
```

Reply:
@JohnStainsby ace, thanks, do we need to sync on this?

Reply:
Yeah would be good, the code I've added to the deployment pipeline is looking for a `.copilot-helper-version` file in the root directory. With the version in x.y.z format.